oecophylla/cli/tests/data/tool_params.yml

```diff
@@ -25,39 +24,29 @@ humann2:
   - cpm
   - relab
   other: ''
-kraken:
-  # kraken_db: /home/qiz173/Databases/Kraken/stdb
-  db: test_data/kraken_test_db
-  kmers: 15
 maxbin: -plotmarker
 metaphlan2:
-  # metaphlan2_dir: /home/jgsanders/git_sw/metaphlan2
-  db: test_data/mp2_test_db/mpa_v20_m200
-
-shogun:
-  db: test_data/shogun_test_db
-  # alignment algorithm: bowtie2, burst and utree
-  aligner: utree
-  # taxonomic ranks (kingdom to strain) to redistribute reads
+  db: test_data/test_dbs/mp2_test_db/mpa_v20_m200
```

Comment:
This `mpa_v20_m200` is the original MetaPhlAn2 database name. We will need to replace it with some minimal thing at some time.